privaterelay/tests/mgmt_sync_phone_related_dates_on_profile_tests.py

```diff
@@ -229,3 +231,44 @@ def test_yearly_phone_subscriber_with_subscription_date_older_than_31_days_profi
     assert profile.date_phone_subscription_start == date_subscribed_phone
     assert profile.date_phone_subscription_end == date_subscribed_phone + timedelta(365)
     assert num_profiles_updated == 1
+
+
+@pytest.mark.django_db
+@patch(f"{MOCK_BASE}.get_phone_subscription_dates")
+def test_command_with_one_update(
+    mocked_dates,
+    patch_datetime_now,
+    phone_user,
+    capsys,
+):
+    profile = Profile.objects.get(user=phone_user)
+    date_subscribed_phone = datetime.now(timezone.utc)
+    profile.date_subscribed_phone = date_subscribed_phone
+    profile.save()
+    mocked_dates.return_value = (
+        date_subscribed_phone,
+        date_subscribed_phone,
+        date_subscribed_phone + timedelta(365),
+    )
+    call_command(SYNC_COMMAND, "--group", "subscription")
+    out, err = capsys.readouterr()
```

Comment:
```suggestion
    out, _ = capsys.readouterr()
```
(suggestion) Since we are not using `err` you can use `_` instead for throwaway/ignore variables.